tests/test_meshtastic_utils.py

```
@@ -755,6 +782,62 @@ def test_connect_meshtastic_invalid_type(self, mock_tcp, mock_ble, mock_serial):
         mock_tcp.assert_not_called()
         mock_ble.assert_not_called()
 
+    @patch("mmrelay.meshtastic_utils.serial_port_exists")
+    @patch("mmrelay.meshtastic_utils.meshtastic.serial_interface.SerialInterface")
+    @patch("mmrelay.meshtastic_utils.meshtastic.ble_interface.BLEInterface")
+    @patch("mmrelay.meshtastic_utils.meshtastic.tcp_interface.TCPInterface")
+    def test_connect_meshtastic_startup_drain_applies_only_once(
+        self, _mock_tcp, _mock_ble, mock_serial, mock_port_exists
```

Ruff (0.15.7)

[warning] Fixture `_mock_tcp` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)